apps/communications/contacts/js/contacts_matching_ui.js

```diff
@@ -10,6 +10,8 @@ var contacts = window.contacts || {};
 if (!contacts.MatchingUI) {
   contacts.MatchingUI = (function() {
 
+    const LEFT_TO_RIGHT_MARK = '\u200E';
```

Comment:
Still think how to resolve this issue in a better way..... The problem is that contact name/intl. number is used inside l10n string. At the moment I can think of the following solutions:

* Redesign this top message so that contact name is rendered in a separate element, like subheader or smth similar - too risky for v2.2. Also we can't just extract contact name variable from l10n string because of the fact that we don't know its correct position in all locales, eg. compare [1] - English, [2] - Polish and [3] - Lithuanian;

* Use Unicode v6.3 isolation marks (FSI/PDI) - Gecko doesn't support it yet (see [4]);

* Use hacky method with LRM mark, if I understand it correctly it shouldn't affect LTR or RTL contact names (if we assume that they start from strong characters), but only "weak" ```+``` sign.

Do you have any better idea?

[1] http://mxr.mozilla.org/l10n-gaia/source/en-US/apps/communications/contacts/matcher/matcher.properties#13
[2] http://mxr.mozilla.org/l10n-gaia/source/pl/apps/communications/contacts/matcher/matcher.properties#10
[3] http://mxr.mozilla.org/l10n-gaia/source/lt/apps/communications/contacts/matcher/matcher.properties#13
[4] https://bugzilla.mozilla.org/show_bug.cgi?id=922963